packages/core/src/config/parse.ts

```diff
@@ -1532,28 +1611,13 @@ permission to call the 'upgradeTo' function on each of them.
 
     // We skip this validation for 'no-proxy' contracts because the checks aren't relevant to them.
     if (contractConfig.kind !== 'no-proxy') {
-      // In addition to doing validation the `getOpenZeppelinUpgradableContract` function also outputs some warnings related to
-      // the provided override options. We want to output our own warnings, so we temporarily disable console.error.
-      const tmp = console.error
-      // eslint-disable-next-line @typescript-eslint/no-empty-function
-      console.error = () => {}
-
-      // fetch the contract and validate
-      // We use a try catch and then rethrow any errors because we temporarily disabled console.error
-      try {
-        upgradableContract = getOpenZeppelinUpgradableContract(
-          contractConfig.contract,
-          input,
-          output,
-          contractConfig.kind,
-          userContractConfig
-        )
-      } catch (e) {
-        throw e
-      }
-
-      // revert to standard console.error
-      console.error = tmp
```

Comment:
Yes, the errors logged in `getOpenZeppelinUpgradableContract` can cause problems in other places, so I decided it would be cleaner to move this logic into `getOpenZeppelinUpgradableContract`.